### PR TITLE
Set up cypress

### LINKED ISSUE #19: End-to-End testing with Cypress
- [x] Set up Cypress testing library
- [x] Set up CI job for testing (@IT2901-SINTEF01/infra)
- [x] Test launch of website
- [x] If applicable; extract test artifacts

### PR DESCRIPTION
**Pull request by blauks**
Closes #19

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
     "scripts": {
         "start": "craco start",
         "build": "craco build",
-        "test": "craco test",
+        "test": "yarn cypress run",
```
Comment (FredrikAugust) [inline package.json]: Do you need to include "yarn" here?

Comment (FredrikAugust) [reply package.json]: I think it will run in the context of yarn by default, ref. `craco`.

Comment (blauks) [reply package.json]: Tested now, I did not need it. Good spot:)

**Comment by FredrikAugust** [comment]
Seems like there might a problem with the caching not caching the cypress installation;

- https://github.com/IT2901-SINTEF01/frontend/pull/24/checks?check_run_id=1904209532#step:5:22
- https://github.com/IT2901-SINTEF01/frontend/pull/24/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR79

**Comment by blauks** [comment]
> Seems like there might a problem with the caching not caching the cypress installation;
> 
>     * https://github.com/IT2901-SINTEF01/frontend/pull/24/checks?check_run_id=1904209532#step:5:22
> 
>     * https://github.com/IT2901-SINTEF01/frontend/pull/24/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR79

That might be it, I will look into it now🤓

File: cypress/support/index.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+// ***********************************************************
```
Comment (FredrikAugust) [inline cypress/support/index.js]: Perhaps remove unnecessary comments?

File: cypress/support/commands.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+// ***********************************************
+// This example commands.js shows you how to
+// create various custom commands and overwrite
+// existing commands.
+//
+// For more comprehensive examples of custom
+// commands please read more here:
+// https://on.cypress.io/custom-commands
+// ***********************************************
```
Comment (FredrikAugust) [inline cypress/support/commands.js]: Do we need this file, or can it be removed?
Perhaps just remove comments if it will be needed later?

**Review by FredrikAugust — DISMISSED** [review]

File: cypress/support/commands.js (hunk 3 above)
Comment (blauks) [reply cypress/support/commands.js]: True! I thought we could have them as a form of "documentation" for later. But I suppose I have google:)

Comment (FredrikAugust) [reply cypress/support/commands.js]: Yep. You can decide if you want to leave it or not! Doesn't make a huge difference, but it's nice to take a stance whether you want to include vendor comments or not.

Comment (blauks) [reply cypress/support/commands.js]: We have to have the file btw, it comes automatically when running a test.

Comment (blauks) [reply cypress/support/commands.js]: But I will remove the comments

**Comment by FredrikAugust** [comment]
Great work 🏎

**Pull request merged**